packages/examples/browser-lsp/src/serverWorker.ts

```diff
@@ -67,22 +66,21 @@ function getColorInformation(textDocument: TextDocumentIdentifier) {
     return colorInfos;
 }
 
-function getColorPresentation(color: Color, range: Range) {
+function getColorPresentation (color: Color, range: Range) {
     const result: ColorPresentation[] = [];
-    const red256 = Math.round(color.red * 255), green256 = Math.round(color.green * 255), blue256 = Math.round(color.blue * 255);
+    const red256 = Math.round(color.red * 255); const green256 = Math.round(color.green * 255); const blue256 = Math.round(color.blue * 255);
```

Comment:
I'm surprised there is no eslint rule preventing putting multiple variables on the same line

Reply:
No, it should not be there. I changed the config to this now (upcoming commit)
```javascript
        'dot-notation': 'off',
        '@typescript-eslint/dot-notation': ['error']
```
but now VSCode is auto-formatting the code to dotted notation all the time and this leads to TypeScript errors, for example:

```shell
src/monaco-language-client.ts:55:103 - error TS2341: Property 'handleRegistrationRequest' is private and only accessible within class 'BaseLanguageClient'.

55         const originalHandleRegistrationRequest: (params: RegistrationParams) => Promise<void> = this.handleRegistrationRequest.bind(this);
```

Reply:
At least it is not configured in `standard`. 🙂